input/docs/fundamentals/criteria.md

```diff
@@ -6,20 +6,113 @@ You can control and influence the flow of the build script execution by providin
 ```csharp
 Task("A")
     .WithCriteria(DateTime.Now.Second % 2 == 0)
-    .Does(() =>
-{
-});
-
+    .Does(() => ... );
```

Comment:
I am not sure about the inclusion of the ... here.  Some might think that this is part of the Cake DSL.  I would prefer that this is left as it was.